src/util/message.h

```diff
@@ -177,13 +184,17 @@ class messaget
     {
     }
 
-    mstreamt(const mstreamt &other):
+    mstreamt(const mstreamt &other)=delete;
+
+    mstreamt(const mstreamt &other, messaget &_message):
```

Comment:
What is it useful for, it's not even tested.

Reply:
Called from `messaget`'s copy constructor.